### DISCUSSION TITLE
Custom messages of rules

### DISCUSSION THREAD

**Discussion by Sharoek** [konnco/filament-import#41] [Q&A] [answered]
How exactly does the custom messages work ? 
In the rules method i see you have defined 2 parameters with rules array and custommessages array. 
As for now i have this set to `->rules(['alpha_num'], ['alpha_num' => 'Only combination of letters and numbers allowed'])`
But in the translation I find `'message' => ':error on line :line',` and this is exactly displayed in the browser. 

I want to display 'Only combination of letters and numbers allowed' when the error occurs. How can i do this?

**Comment by frankyso** [comment, marked as answer]
Hi @Sharoek please have a check on the latest version :)
